packages/react/slider/src/Slider.tsx

```diff
@@ -92,56 +92,38 @@ const Slider = React.forwardRef((props, forwardedRef) => {
   const [values = [], setValues] = useControllableState({
     prop: value,
     defaultProp: defaultValue,
-    onChange: onValueChange,
+    onChange: (value) => {
+      const thumbs = [...thumbRefs.current];
+      thumbs[valueIndexToChangeRef.current]?.focus();
+      onValueChange(value);
+    },
   });
 
   function handleSlideStart(value: number) {
     const closestIndex = getClosestValueIndex(values, value);
-    updateValues(value, closestIndex).then((valueIndexToChange) => {
-      /**
-       * Browsers fire event handlers before executing their event implementation
-       * so they can check if `preventDefault` was called first. Therefore,
-       * if we focus the thumb on slide start (`mousedown`), the browser will execute
-       * their `mousedown` implementation after our focus which will instantly
-       * `blur` the thumb again (because it effectively clicks off the thumb).
-       *
-       * We use a `setTimeout`  to move the focus to the next tick (after the
-       * mousedown) to ensure focus on mousedown.
-       */
-      window.setTimeout(() => focusThumb(valueIndexToChange), 0);
```

Comment:
So good that we can remove this! Were you able to check the regression on this one? I can't remember myself how to check for it.

Reply:
Yeah, to test you have to click somewhere on the track and the thumb should focus.

Reply:
That was the one! Thanks! 🙏